details.yml

```diff
@@ -47,7 +47,7 @@ education:
   city: Naumburg
 
 # Settings
-mainfont: Hoefler Text
```

Comment:
I couldn't find where to download this font since Hoefler Text is only available in MacOS, and Docker runs Linux. So instead opted to using `Crimson Text` font, which I think is the closest to the original font